apps/web/modules/bookings/views/bookings-listing-view.tsx

```diff
@@ -214,7 +214,7 @@ function BookingsContent({ status }: BookingsProps) {
             ),
             isToday: true,
           }))
-      )[0] || []
```

Comment:
Only getting bookings from the first page of this map was not causing all bookings to be rendered